grunt/grunt-generate-doc.js

```diff
@@ -334,10 +334,14 @@ exports.generateDoc = function generateDoc({files, targetPath, version}) {
   }
 
   function renderTypeLink(name) {
-    if (!typeLinks[name]) {
-      return name;
-    }
-    return `[${name}](${typeLinks[name]})`;
+    let types = [];
+    name.split('|').forEach((name) => {
+      if (typeLinks[name]) {
+        types.push(`[${name}](${typeLinks[name]})`);
+      }
+      types.push(name);
+    });
+    return types.join('\\|');
```

Comment:
I think this could be simplified a bit by using `map()` instead of `forEach()`?
```js
return name.split('|')
  .map(name => typeLinks[name] ? `[${name}](${typeLinks[name]})` : name)
  .join('\\|');
```